x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/expandable_section/expandable_section_analytics.tsx

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useEffect, useState, FC } from 'react';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { EuiHorizontalRule, EuiLoadingSpinner, EuiSpacer, EuiText } from '@elastic/eui';
+
+import type { DataFrameAnalysisConfigType } from '../../../../../../../common/types/data_frame_analytics';
+
+import { ml } from '../../../../../services/ml_api_service';
+
+import { getAnalysisType } from '../../../../common';
+
+import { isGetDataFrameAnalyticsStatsResponseOk } from '../../../analytics_management/services/analytics_service/get_analytics';
+import {
+  DataFrameAnalyticsListRow,
+  DATA_FRAME_MODE,
+} from '../../../analytics_management/components/analytics_list/common';
+import { ExpandedRow } from '../../../analytics_management/components/analytics_list/expanded_row';
+
+import { ExpandableSection, ExpandableSectionProps } from './expandable_section';
+
+const getAnalyticsSectionHeaderItems = (
+  expandedRowItem: DataFrameAnalyticsListRow | undefined
+): ExpandableSectionProps['headerItems'] => {
+  return expandedRowItem !== undefined
+    ? [
+        {
+          id: 'analysisTypeLabel',
+          label: (
+            <FormattedMessage
+              id="xpack.ml.dataframe.analytics.exploration.analysisTypeLabel"
+              defaultMessage="Type"
+            />
+          ),
+          value: expandedRowItem.job_type,
+        },
+        {
+          id: 'analysisSourceIndexLabel',
+          label: (
+            <FormattedMessage
+              id="xpack.ml.dataframe.analytics.exploration.analysisSourceIndexLabel"
+              defaultMessage="Source index"
+            />
+          ),
+          value: expandedRowItem.config.source.index,
+        },
+        {
+          id: 'analysisDestinationIndexLabel',
+          label: (
+            <FormattedMessage
+              id="xpack.ml.dataframe.analytics.exploration.analysisDestinationIndexLabel"
+              defaultMessage="Destination index"
+            />
+          ),
+          value: expandedRowItem.config.dest.index,
+        },
+      ]
+    : 'loading';
```

Comment:
Small suggestion to use a constant for this `loading` since it's used across several files.

Reply:
Good point, uses a const now in ccb7d9b.